Sources/Reports.php

```diff
@@ -319,38 +321,24 @@ function BoardPermissionsReport()
 
 	// Fetch all the board names.
 	$request = $smcFunc['db_query']('', '
-		SELECT id_board, name, id_profile
-		FROM {db_prefix}boards
+		SELECT id_board, b.name, id_profile
+		FROM {db_prefix}categories AS c
+			JOIN {db_prefix}boards AS b USING (id_cat)
 		WHERE ' . $board_clause . '
-		ORDER BY id_board',
+		ORDER BY cat_order, child_level, board_order',
```

Comment:
same here would add alias,
for better understand from where the column is coming